deblur/workflow.py

```diff
@@ -41,9 +47,95 @@ def dereplicate_seqs(seqs_fp,
         log_name=log_name)
 
 
-def remove_artifacts_seqs(seqs_fp):
-    """Step 3: remove artifacts from FASTA file"""
-    pass
+def remove_artifacts_seqs(seqs_fp,
+                          ref_fp,
+                          output_fp,
+                          ref_db_fp=None,
+                          negate=False,
+                          threads=1):
+    """Step 3: remove artifacts from FASTA file using
```

Comment:
Correct, that was an oversight from the previous reviews. The functions should not have anything related with the step that they place in the workflow. If in the future we do some change in the pipeline and we add some step, those numbers are no longer correct, which can lead to confusion. Having the function descriptions being agnostic to the step they place in the workflow will save that error.